app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -113,26 +127,32 @@ public function storeUrl(StoreIntegrationUrlRequest $request, string $id): Redir
         );
     }
 
-    public function destroy(Request $request, string $id): RedirectResponse
+    public function show(string $id): Response
     {
         try {
-            $this->integrationRepository->deleteById(Uuid::fromString($id));
-        } catch (ModelNotFoundException) {
-            // We can redirect back to integrations, even if not successful
+            $integration = $this->integrationRepository->getById(Uuid::fromString($id));
+            $subscription = $this->subscriptionRepository->getById($integration->subscriptionId);
+            $contacts = $this->contactRepository->getByIntegrationId(UUid::fromString($id));
+        } catch (Throwable) {
+            abort(404);
         }
 
-        return Redirect::route(
-            TranslatedRoute::getTranslatedRouteName($request, 'integrations.index')
-        );
+        return Inertia::render('Integrations/Detail', [
+            'integration' => [
+                ...$integration->toArray(),
```

Comment:
This is a single item, not an array, so no need for a splat operator I think?

Reply:
The splat operator is used to use all the key value pairs from the integration and add more to it

e.g.

```php
$userInfo = [
    'name' => 'John Doe',
    'age' => 27,
];

$splatted = [
    ...$userInfo,
    'isAdmin' => false
];

var_dump($splatted);
```

Reply:
I see now, it is in the context of the request!